components/AboutMembers/index.tsx

```diff
@@ -3,26 +3,36 @@ import Header from '../Header';
 import { colors } from '../../stylesheets/colors';
 import styled from 'styled-components';
 import { MEMBERS } from '../../constants/about';
-import { BaseSection } from '../BaseSection';
 import LinkText from '../LinkText';
 import Link from 'next/link';
 
 const MemberLink = styled(LinkText)`
   text-transform: uppercase;
+  font-size: 1.8em;
 `;
 
-const Section = styled(BaseSection)`
-  height: 100vh;
-  background-color: ${colors.backgroundColor2};
-  padding: 100px 0;
+const Section = styled.div`
+  min-height: 100vh;
+  padding: 80px 0;
   box-sizing: border-box;
 `;
 
+const CustomHeader = styled(Header)`
+  color: ${(props) => props.color};
+  @media (prefers-color-scheme: dark) {
+    color: ${colors.paragraphColor};
+  }
```

Comment:
Instead of defining media queries for color everywhere we could define the dark colors as an alternate color-scheme in the global color declaration.

Like so:
``` css
/* global.css */
--light: #fff;
--dark: #000;

--textColor: var(--light);
@media (prefers-color-scheme: dark) {
  --textColor: var(--dark);
}

/* Component.tsx */
color: var(--textColor);
```

What this gives us is that the dark and light colors are only defined one place.
Every light color has a corresponding dark color, and you don't have do care about dark and light mode in each component :smile: